kmip/core/config_helper.py

```diff
@@ -35,6 +35,7 @@ class ConfigHelper(object):
     DEFAULT_SSL_VERSION = 'PROTOCOL_SSLv23'
     DEFAULT_USERNAME = None
     DEFAULT_PASSWORD = None
+    DEFAULT_TIMEOUT = 30
```

Comment:
Yep, I'll add one. It's seconds, `socket.settimeout` only accepts seconds as a float. I guess we could use milliseconds and manually convert up, if we really wanted to.